test/integration/addons_test.go

```diff
@@ -36,7 +36,6 @@ import (
 
 // TestAddons tests addons that require no special environment -- in parallel
 func TestAddons(t *testing.T) {
-	MaybeParallel(t)
```

Comment:
Can we take this line out for now? It's too difficult to see if this PR makes the parallel integration testing behavior better or worse with it.

Reply:
yup, it should be out!